src/app/components/Embeds/OEmbed/index.tsx

```diff
@@ -16,19 +20,53 @@ type OEmbedData = Partial<{
   width: number;
   height: number;
   type: string;
+  parameters: ampParams;
+  oEmbedType: string;
 }>;
 
 export type OEmbedProps = {
   type: string;
   oembed: OEmbedData;
 };
 
+export type ampParams = {
+  'amp-clickable': boolean;
+  'amp-image-height': number;
+  'amp-image-width': number;
+  'amp-image': string;
+};
+
+export type OEmbedAmpProps = {
+  isVDJEmbed: boolean;
+  canonicalLink: string;
+  parameters?: ampParams;
+  url?: string;
+};
+
 const OEmbedLoader = ({ oembed }: OEmbedProps) => {
-  const { translations } = useContext(ServiceContext);
   const { isAmp, canonicalLink } = useContext(RequestContext);
-  const { html } = oembed;
+  const { translations } = useContext(ServiceContext);
+  const { html, oEmbedType, parameters, url } = oembed;
+  const isVDJEmbed = oEmbedType === 'vdj-embed';
 
   if (isAmp) {
+    if (isVDJEmbed) {
+      if (parameters) {
```

Comment:
I've included this so that we don't render nothing at the URL http://localhost:7080/pidgin/articles/c0nzj66vp65o.amp?renderer_env=test

An alternative would be to have `if (!parameters) return null`. As I understand parameters will always be supplied. soo we could potentially remove this once the BFF changes are merged in.